test/shopify_app/configuration_test.rb

```diff
@@ -209,4 +209,14 @@ class ConfigurationTest < ActiveSupport::TestCase
     assert_equal ShopifyApp::AccessScopes::ShopStrategy, ShopifyApp.configuration.shop_access_scopes_strategy
     assert_equal ShopifyApp::AccessScopes::UserStrategy, ShopifyApp.configuration.user_access_scopes_strategy
   end
+
+  test "user access scopes strategy is configurable" do
+    my_strategy = Object
+    ShopifyApp.configure do |config|
+      config.user_access_scopes_strategy = my_strategy
```

Comment:
We set values such as `user_session_repository` using strings rather than constants, I wonder if need to that here too... 🤔

Reply:
Updated to use a string.